lib/LaTeXML/Package/LaTeX.pool.ltxml

```diff
@@ -4981,7 +4996,10 @@ DefMacro('\@secondoftwo{}{}',    sub { $_[2]; });
 DefMacro('\@thirdofthree{}{}{}', sub { $_[3]; });
 DefMacro('\@expandtwoargs{}{}{}', sub {
     ($_[1]->unlist, T_BEGIN, Expand($_[2])->unlist, T_END, T_BEGIN, Expand($_[3])->unlist, T_END); });
-DefMacro('\@makeother{}', sub { AssignCatcode($_[1] => CC_OTHER, 'local'); });
+DefMacro('\@makeother{}', sub {
+    my ($gullet, $char) = @_;
+    $char = ToString($char); $char =~ s/\\//;
```

Comment:
I guess removing the leading backslash is really all that is needed... Fingers crossed

Reply:
The TeX book is rather confusing. One form of "normal integer" is backquote followed by a "character token". The latter (which is not expanded) is defined as  " a (character code, category code) pair, or an active character, or a control sequence whose name consists of a single character."  I have no idea what the pair thing is about (no syntax is specified). The rest sounds like you should just give the character if it's active, otherwise precede it with a backslash.  However, in simple tests using pdflatex, I get the same thing for active and non-active both with and without the backslash.

Reply:
It brings me no pleasure to do this, but I sadly managed to get a torture case which would fail this perly implementation. Here it is:

```tex
\documentclass{article}
\begin{document}
\makeatletter
\catcode`\%=0
\@makeother\\
test \\ here
%end{document}
```

Reply:
And now that I've taken a step back -- I am wondering whether this calls for a new `DefParameterType` that deals with what is supposedly a common situation in a number of macros?

Reply:
latexml & pdflatex are same here.

Reply:
Ha! I should've tested against the right branch... It does indeed work. We'll see if arXiv can break it next :> Good to see you've merged already.